js/libs/canvas2d.js

```diff
@@ -16,9 +16,19 @@ class C2D {
     canvas.height = height || window.innerHeight
     document.body.appendChild(canvas)
 
+    this._fontOptions = {
+      size: 48,
+      style: 'serif',
+      align: 'center',
+      baseline: 'middle'
+    }
+
```

Comment:
great use of the underscore `_` here! other languages allow u to create "private" properties (ie. properties which are only accessible internally by the class/object && not externally when using it)... sadly JavaScript does not (all properties act as "public" properties), that said, the JS community has established a convention of adding an underscore to properties that are meant to be private, to signal to other coders using the library that they should try to avoid interacting directly with it.

i would change the name though, i think "options" here is sort of implied, so to keep the code a little tighter, i'd just call this `this._font`